build.gradle.kts

```diff
@@ -110,33 +48,42 @@ val packageVersion: String by project
 group = pluginGroup
 version = packageVersion
 
-logger.lifecycle("Building Amazon Ion $pluginVersion for ${descriptor.platformType} ${descriptor.platformVersion}")
+logger.lifecycle("Building Amazon Ion $pluginVersion")
 
 dependencies {
     intellijPlatform {
-        create(descriptor.platformType.acronym, descriptor.platformVersion) // https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin.html#setting-up-intellij-platform
-        bundledPlugins(descriptor.bundledDependencies)
+        // This should be equivalent to the lowest version that we build against.
+        intellijIdeaCommunity(supportedSinceIdeVersion.let(::branchToProductReleaseVersion))
+
+        bundledPlugins(listOf(
+            "com.intellij.java",
+            "org.jetbrains.kotlin",
+        ))
         pluginVerifier()
         instrumentationTools()
 
         testFramework(TestFrameworkType.Platform)
     }
 
     testImplementation("org.junit.jupiter:junit-jupiter:5.7.1")
-    testCompileOnly("junit:junit:4.13")
+    testCompileOnly("junit:junit:4.13.1")
     testRuntimeOnly("org.junit.vintage:junit-vintage-engine")
 }
 
 intellijPlatform {
+    // Disable searchable options since it is failing during the build
+    // Re-evaluate in the future if it starts to succeed.
+    buildSearchableOptions = false
```

Comment:
Thanks for the comment! Can you add to this a notion of what signal to look for?

Reply:
This was something that existed prior to my involvement, and I just left it alone. However, testing it locally seems to work just fine without this line, so I'll remove it.